U/womullan/pst cookie (#138)
lsst/lsst-texmf: docs/templates/document.rst

This section describes configurations requested by :command:`cookiecutter`.

``org``

>    Which part of DM the note is from (this comes out on the title page:

Reviewer: ```suggestion
   Which part of DM the note is from (this comes out on the title page):
```

Reviewer: Actually, most of these aren't “part of DM”. How about:
```suggestion
   Which part of the LSST project the note is from (this is displayed on the title page):
```

   - PST - Project Science Team
   - DM  - Data Management
   - SE  - Systems Engineering
   - PMO - Project Office

``series``
   Handle of the documentation series.

>    Technical notes can be ``DMTN``, ``SQR``, ``PSTN'' or ``SMTN`` (see the `DM Developer Guide <https://developer.lsst.io/docs/technotes.html>`__ for more information).

Reviewer: ```suggestion
   Technical notes can be ``DMTN``, ``SQR``, ``PSTN`` or ``SMTN`` (see the `DM Developer Guide <https://developer.lsst.io/docs/technotes.html>`__ for more information).
```
